scripts/west_commands/twister_cmd.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2022 Google
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+from west import log
+
+from zephyr_ext_common import Forceable
+from zephyr_ext_common import get_twister_path
+
+# Resolve path to twister libs and add imports
+twister_path = get_twister_path()
+os.environ["ZEPHYR_BASE"] = str(twister_path.parent)
+
+sys.path.insert(0, str(twister_path))
+sys.path.insert(0, str(twister_path / "pylib" / "twister"))
+
+from importlib.util import spec_from_loader, module_from_spec
+from importlib.machinery import SourceFileLoader
+
+spec = spec_from_loader(
+    "twister", SourceFileLoader("twister", str(twister_path / "twister"))
+)
+twister = module_from_spec(spec)
+spec.loader.exec_module(twister)
+
+from twisterlib.environment import add_parse_arguments, parse_arguments
+
+BUILD_DESCRIPTION = """\
+Convenience wrapper for twister.
+"""
```

Comment:
I got around to trying this out today. The help output is much improved since not overriding the default, but I think we can do a little better here:

```
"""\
Convenience wrapper for twister. The below options are shared
with the twister script and have the same effects as if you ran twister
directly. Refer to the twister documentation for more information.
"""